README.md

```diff
@@ -331,6 +331,85 @@ personalisation={
 
 </details>
 
+### Precompiled Letter
+
+#### Method
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```python
+response = notifications_client.send_precompiled_letter_notification(
+    reference,      # Reference to identify the notification
+    # content or pdf_file is required
+    content=None,   # base64 encoded string
+    pdf_file=None   # pdf file object
+)
+```
+</details>
+
+#### Response
+
+If the request is successful, `response` will be a `dict`. 
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```python
+{
+  "id": "740e5834-3a29-46b4-9a6f-16142fde533a",
+  "reference": "your-letter-reference",
+  "content": {
+    "subject": "Licence renewal",
+    "body": "Dear Bill, your licence is due for renewal on 3 January 2016.",
+  },
+  "uri": "https://api.notifications.service.gov.uk/v2/notifications/740e5834-3a29-46b4-9a6f-16142fde533a",
+  "template": {
+    "id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+    "version": 1,
+    "uri": "https://api.notifications.service.gov.uk/v2/template/f33517ff-2a88-4f6e-b855-c550268ce08a"
+  }
+  "scheduled_for": None
+}
+```
+
+Otherwise the client will raise a `HTTPError`:
+
+|`error.status_code`|`error.message`|
+|:---|:---|
+|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type live of 10 requests per 20 seconds"`<br>`}]`|
+|`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send letters with a team api key"`<br>`]}`|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send precompiled letters"`<br>`]}`|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send letters when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "reference is a required property"`<br>`}]`|
```

Comment:
should include an invalid pdf error in here

Reply:
Not sure what that will look like at the moment, as there's no checking of the File obj passed on API, so we are concentrating on the happy path.